docs/website/netcdf-java/reference/stream/CdmrCommon.adoc

```diff
@@ -1,12 +1,12 @@
 CDM Remote is a web service implemented in the CDM library (client) and TDS (server), providing remote access to CDM datasets, using
-<<NcStream#,ncstream>> as the on-the-wire protocol.
+link:NcStream.adoc[ncstream] as the on-the-wire protocol. An experimental python client is under development.
```

Comment:
This should remain a cross-reference. That way, when asciidoctor generates the HTML, the link will point to `NcStream.html`, not `NcStream.adoc`.